variables.tf

```diff
@@ -104,4 +118,27 @@ variable "ec2_security_group_ids" {
   type    = list(string)
   default = []
   description = "List of security group IDs to attach. If empty, the default SG is used."
-}
\ No newline at end of file
+}
+
+variable "permission_boundary" {
+  type        = bool
+  description = "Enable permission boundary for the IAM role."
+  default     = false
+}
+
+variable "env_name" {
```

Comment:
env_name is available from the cert (pem var)

Reply:
done